internal/pkg/cli/svc_init.go

```diff
@@ -288,7 +289,7 @@ func (o *initSvcOpts) Execute() error {
 		}
 	}
 	// If the user passes in an image, their docker engine isn't necessarily running, and we can't do anything with the platform because we're not building the Docker image.
-	if o.image == "" {
+	if o.image == "" && !o.manifestExists {
```

Comment:
Does Copilot today assumes if manifest is provided then image.location has to be there?

Reply:
Given user input on image, dockerfile, etc., `svc init` creates a manifest for users to start with. I do not feel like it's a requirement that they have to have provided an image during `svc init`.